src/views/Feedback/index.js

```diff
@@ -78,10 +78,7 @@ export default class Feedback extends Component {
                     marginRight: 'auto',
                   }}
                 >
-                  <img
-                    src={require(`${'../../NoConnection.svg'}`)}
-                    alt="Internet Connection Lost"
-                  />
+                  <NoConnection />
```

Comment:
Perhaps change to NoConnectionSVG for readability
```suggestion
                  <NoConnectionSVG />
```

Reply:
And the same for all of the others :)